perl/README.md

```diff
@@ -0,0 +1,26 @@
+# Bystro Annotator
+
+## Install
+
+```bash
+# system install
+cpanm Dist::Zilla
```

Comment:
Need 

`dzil authordeps | cpanm`

before `dzil install` or install will fail. If I just read the instructions, I'm not sure what to do with Author dependencies, and comes after the installation instructions, so I wouldn't think to run it first.

We also want `cpanm Archive::Tar::Wrapper`

to get rid of the warning `[DZ] building archive with Archive::Tar; install Archive::Tar::Wrapper 0.15 or newer for improved speed` and improve build speed